Sources/System/Internals/Syscalls.swift

```diff
@@ -101,7 +101,7 @@ internal func system_pwrite(
 #if ENABLE_MOCKING
   if mockingEnabled { return _mockInt(fd, buf, nbyte, offset) }
 #endif
-  return pwrite(fd, buf, nbyte, offset)
+  return pwrite(fd, buf!, nbyte, offset)
```

Comment:
I'm afraid we can't do it this way. This pointer comes from a caller-supplied buffer, whose `baseAddress` could well be `nil`. For example:
```
    let fd = FileDescriptor.standardOutput
    let empty = UnsafeRawBufferPointer(start: nil, count: 0)
    let written = try fd.write(toAbsoluteOffset: 0, empty) // crash at runtime with the suggested force-unwrap
```
We don't have that precise case covered in a test, but we should!

Reply:
OK, I will check it instead.